test/SparqlAntlrParserTestHelpers.h

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Robin Textor-Falconi (textorr@informatik.uni-freiburg.de)
+
+#pragma once
+
+#include <gmock/gmock.h>
+
+#include "../src/parser/data/VarOrTerm.h"
+
+template <class>
+constexpr bool always_false_v = false;
+

```

Comment:
// Not relevant for the actual test logic, but provides human-readable output if a 
// test fails.